src/System.Management.Automation/engine/parser/Compiler.cs

```diff
@@ -1807,7 +1807,15 @@ private static RuntimeDefinedParameter GetRuntimeDefinedParameter(ParameterAst p
             for (int index = 0; index < parameterAst.Attributes.Count; index++)
             {
                 var attributeAst = parameterAst.Attributes[index];
-                var attribute = attributeAst.GetAttribute();
+                Attribute attribute;
+                try
+                {
+                    attribute = attributeAst.GetAttribute();
+                }
+                catch (RuntimeException)
+                {
+                    continue;
```

Comment:
Since it is Compiler can this change a behavior of real script execution?

Reply:
I don't know. We need someone that is more familiar with the engine to answer that question.